portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase

```diff
@@ -1121,7 +1058,7 @@ definition {
 				portalVersionOutput = "${portalVersionOutput}");
 		}
 
-		task ("View OAuth application scope") {
+		task ("View OAuth2 application scope") {
```

Comment:
I think we should be consistent: is OAuth 2 and application name or just type indicator, later you use the wording "Oauth 2" but here it's "OAuth2"

Reply:
You are right, I switched **OAuth2** to **OAuth 2** everywhere in the test case file.